### PR TITLE
feat: 3753/ma sign in copy

### PR DESCRIPTION
**Pull request by cade-exygy**
# Pull Request Template

## Issue Overview

This PR addresses #3753 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added Sign In/Up benefits copy the existing sign in page. 

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.
- Set the SHOW_MANDATED_ACCOUNTS env to TRUE
- Go to the Sign In Page
- Ensure styles match with Figma and mobile responsiveness is working properly

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by ColinBuyck** [comment]
@cade-exygy Heads up that this isn't building! Let me know if you have any questions on resolving that 🦞

File: sites/public/src/pages/sign-in.tsx
Comment (cade-exygy) [inline sites/public/src/pages/sign-in.tsx:116]: These changes are to fix a type error that was preventing the build.

**Comment by cade-exygy** [comment]
https://github.com/cypress-io/cypress/issues/8970

**Comment by emilyjablonski** [comment]
@cade-exygy Just pulling over my same comment from the other PR, would love to see green tests before a full review, lmk if you would like to pair!

**Comment by cade-exygy** [comment]
The failing tests are caused by the duplicate forms in DOM, due to how I achieved the mobile styling changes. So I am looking into changing how I am handling mobile responsiveness, any ideas on that are welcome!

File: .circleci/config.yml
Comment (emilyjablonski) [inline .circleci/config.yml:33]: Just a reminder based on our last Slack convo that since we expect this toggle will remain off in core, we can't turn it on in the main suite. We will need a separate suite like we have for listings-approval.

File: shared-helpers/src/locales/general.json
Comment (emilyjablonski) [inline shared-helpers/src/locales/general.json:15]: Would you mind alphabetizing these?

File: sites/public/src/components/account/SignUpBenefits.tsx
Comment (emilyjablonski) [inline sites/public/src/components/account/SignUpBenefits.tsx:15]: Similar callout from Slack convo that this is all good to keep for now since the work is already finished, but moving forward to not use tailwind styles but apply scss in module files.

File: sites/public/src/pages/sign-in.tsx
Comment (emilyjablonski) [inline sites/public/src/pages/sign-in.tsx:131]: There's some repetition here, could we (1) name this something more specific and (2) make it a function that we pass a flag to that changes just the class names?

File: sites/public/src/pages/sign-in.tsx
Comment (emilyjablonski) [inline sites/public/src/pages/sign-in.tsx:160]: This block is repeated below, can we define it above once?

**Comment by emilyjablonski** [comment]
A few visual notes!
* There is more margin between the form card and the content in the designs than in the PR
* There is more margin between the icons and the text in the designs than in the PR
* The font size of the heading on the content section is larger in the designs than in the PR

And more of a question, I was surprised to see the content header be sans-serif, have you confirmed that w Jesse or Em?

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Some notes above!

File: sites/public/src/pages/sign-in.tsx
Comment (cade-exygy) [reply sites/public/src/pages/sign-in.tsx:160]: I had some odd keyboard focus issues as a result of doing something similar before. I can take another look. I do like the env var toggle simplicity, but I could also pull the classNames into a module and toggle those based on env var, though the DOM structure for the default view may have a extra div elements

**Comment by cade-exygy** [comment]
> A few visual notes!
> 
> * There is more margin between the form card and the content in the designs than in the PR
> * There is more margin between the icons and the text in the designs than in the PR
> * The font size of the heading on the content section is larger in the designs than in the PR
> 
> And more of a question, I was surprised to see the content header be sans-serif, have you confirmed that w Jesse or Em?

Correct me if I am mistaken, but it looks like the font styles are coming from seeds. Do we want to override?
Screenshot 2024-01-24 at 2 27 00 PM

File: sites/public/src/pages/sign-in.tsx
Comment (cade-exygy) [reply sites/public/src/pages/sign-in.tsx:152]: Yeah I ran into those keyboard issues again. Leaving for now, open for ideas!

**Comment by emilyjablonski** [comment]
@cade-exygy So sorry not following your latest question, the weights looked good to me!

**Comment by cade-exygy** [comment]
> @cade-exygy So sorry not following your latest question, the weights looked good to me!

No worries! Do you see any other visual changes I need to make?

File: sites/public/src/pages/sign-in.tsx
Comment (cade-exygy) [reply sites/public/src/pages/sign-in.tsx:152]: @emilyjablonski @ColinBuyck Just clarifying that I plan on leaving this as is for now. Let me know your thoughts!

**Review by ColinBuyck — APPROVED** [review]
LGTM 🦞  just a couple notes:
1) The top padding of the top copy section ("Sign up quickly...) on mobile view should be 24 px based on the Figma. I believe the bottom should be changed to that as well but I left a comment in the Figma to confirm
2) I'm noticing a lot of flex grow, flex shrink, and order calls still in the changes. I checked a couple of the cases and didn't see how they were utilized. Can any of those be cleaned up or are they all still needed?

File: sites/public/src/pages/sign-in.tsx
Comment (ColinBuyck) [reply sites/public/src/pages/sign-in.tsx:152]: Would you be able to screen record or describe the keyboard focus issues more? Curious to see if its something we've encountered before and may have a known solution for

Comment (emilyjablonski) [reply sites/public/src/pages/sign-in.tsx:152]: Same, would love to pair!

Comment (cade-exygy) [reply sites/public/src/pages/sign-in.tsx:152]: Screenshot 2024-01-30 at 10 01 46 AM

https://github.com/bloom-housing/bloom/assets/131277283/682b1e28-bd75-4bfd-9cd5-8f06cbe45300

Here is what I am seeing when I pull the SignIn code into a component. Let me know if I am missing something!

**Comment by cade-exygy** [comment]
> LGTM 🦞 just a couple notes:
> 
> 1. The top padding of the top copy section ("Sign up quickly...) on mobile view should be 24 px based on the Figma. I believe the bottom should be changed to that as well but I left a comment in the Figma to confirm
> 2. I'm noticing a lot of flex grow, flex shrink, and order calls still in the changes. I checked a couple of the cases and didn't see how they were utilized. Can any of those be cleaned up or are they all still needed?

Can you explain the cases that you noticed? I will see if I can find any redundant ones, but some are needed

File: sites/public/src/pages/sign-in.tsx
Comment (ColinBuyck) [inline sites/public/src/pages/sign-in.tsx:168]: @cade-exygy At first glance, I couldn't see how the grow and shrink values were being utilized on the line above. Haven't used them before though so I could definitely be missing it! Just know that grow is defaulted to zero so maybe that can be removed? Trust your call here!

Comment (cade-exygy) [reply sites/public/src/pages/sign-in.tsx]: Cleaned those styles up! Lmk if there is anything else!

**Review by emilyjablonski — APPROVED** [review]
🥳

**Pull request merged**
